app/src/main/AndroidManifest.xml

```diff
@@ -13,18 +13,32 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Wizard"
-        tools:targetApi="31"
         android:usesCleartextTraffic="true"
-        >
+        tools:targetApi="31">
         <activity
-            android:name=".MainActivity"
+            android:name=".activities.LoginActivity"
             android:exported="true">
             <intent-filter>
```

Comment:
We still want to open the `MainActivity` whenever we click on the app. If we implement the authentication check and storage of the authentication in a separate ticket, I would recommend adding a simple login button to the `MainActivity` for testing purposes.